### DISCUSSION TITLE
Ubuntu 22.10 PPA fails

### DISCUSSION THREAD

**Discussion by g7kse** [foldynl/QLog#165] [Q&A] [answered]
Hi,

I get an error...
`The repository 'https://ppa.launchpadcontent.net/foldyna/qlog/ubuntu kinetic Release' does not have a Release file.`
when trying to install. Is this something I haven't done correctly?

Alex

**Comment by foldynl** [comment, marked as answer]
Hi Alex,

My plan was to prepare QLog only for LTS Ubuntu versions (currently 20.04 and 22.04). But in view of the fact that it is easy to expand the version list, I have prepared a package for 22.10. 

Try to install it again.

**Reply by g7kse** [reply]
Brilliant, thanks. Installed and running nicely
